File: framework/module_common/src/main/java/com/xxl/hello/common/config/CacheDirConfig.java
package com.xxl.hello.common.config;

import com.xxl.kit.PathUtils;

import java.io.File;

/**
 * @Description 缓存文件夹配置
 * @Author: xxl
 * @Date: 2021/8/29 12:40 AM
 **/
public class CacheDirConfig {

    /**
     * 缓存文件夹
     */
    public static final String CACHE_DIR = PathUtils.getAppExtCachePath();

    /**
     * 图片压缩缓存文件夹
     */
    public static final String COMPRESSION_FILE_DIR = PathUtils.getAppExtCachePath() + "/.compression_file_dir";

    /**
     * 分享缓存文件夹
     */
    public static final String SHARE_FILE_DIR = PathUtils.getExtStoragePath() + File.separator + "hello_share";

    /**
     * Tbs缓存文件夹
     */
    public static final String TBS_CACHE_DIR = PathUtils.getAppExtCachePath() + "/.tbs_cache_dir";
}

File: framework/module_common/src/main/java/com/xxl/hello/common/config/CacheDirConfig.kt
package com.xxl.hello.common.config

import com.xxl.kit.PathUtils
import java.io.File

/**
 * 缓存文件夹相关
 *
 * @author xxl.
 * @date 2023/3/2.
 */
class CacheDirConfig private constructor() {

    companion object {

        /**
         * 缓存文件夹
         */
        @JvmField
        val CACHE_DIR = PathUtils.getAppExtCachePath()

        /**
         * 图片压缩缓存文件夹
         */
        @JvmField
        val COMPRESSION_FILE_DIR = PathUtils.getAppExtCachePath() + "/.compression_file_dir"

        /**
         * 分享缓存文件夹
         */
        @JvmField
        val SHARE_FILE_DIR = PathUtils.getExtPicturesPath() + File.separator + "hello_share"

        /**
         * Tbs缓存文件夹
         */
        @JvmField
        val TBS_CACHE_DIR = PathUtils.getAppExtCachePath() + "/.tbs_cache_dir"
    }
}